CHANGELOG.md

```diff
@@ -11,6 +11,12 @@
   [Rounak Jain](https://github.com/rounak)
   [#2028](https://github.com/realm/SwiftLint/issues/2028)
 
+* Add `all` keyword for use in disable / enable statement:
+  `// swiftlint:disable all`.
+  It allows disabling SwiftLint entirely for a particular section.
```

Comment:
Don't forget to include two trailing spaces at the end of the description (check the other entries on the ChangeLog).

Reply:
Done. I originally did so, but Xcode dropped it automatically ;( Used another editor now